### PR TITLE
[Dev docs] How to develop with Texture

### PR DESCRIPTION
**Pull request by nguyenhuy**
Quick guide on how to start developing with Texture.

### CODE REVIEW

File: docs/_docs/development/how-to-develop.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: How to start contributing to Texture
+layout: docs
+permalink: /development/how-to-develop.html
+---
+
+As an open source project, contributions are always welcome. Before you start, please read our <a href = "https://github.com/TextureGroup/Texture/blob/master/CONTRIBUTING.md">Contribution Guidelines</a>. It's als a good idea to familiar yourselves with our <a href = "overview.html">Development documentations</a>.
```
Comment (mikezucc) [inline docs/_docs/development/how-to-develop.md]: It's *also* a good idea to *familiarize* *yourself* with our

File: docs/_docs/development/how-to-develop.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: How to start contributing to Texture
+layout: docs
+permalink: /development/how-to-develop.html
+---
+
+As an open source project, contributions are always welcome. Before you start, please read our <a href = "https://github.com/TextureGroup/Texture/blob/master/CONTRIBUTING.md">Contribution Guidelines</a>. It's also a good idea to familiarize yourself with our <a href = "overview.html">Development documentations</a>.
+
+Setting up your dev environment:
+- If you don't have CocoaPods installed on your machine yet, you should install it now.
```
Comment (maicki) [inline docs/_docs/development/how-to-develop.md]: Add a link to the documentation how to install?

File: docs/_docs/development/how-to-develop.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- Clone the framework's source code to your machine. More details can be found <a href = "https://help.github.com/en/articles/cloning-a-repository">here</a>.
```
Comment (maicki) [inline docs/_docs/development/how-to-develop.md]: Let's add a code example how to do it.

**Comment by nguyenhuy** [comment]
Thanks everyone for reviewing. Let's land this diff and iterate if need to.

**Pull request merged**